haproxy/values.yaml

```diff
@@ -0,0 +1,32 @@
+replicaCount: 1
+
+image:
+  repository: docker.io/haproxytech/haproxy-debian
+  tag: 2.1
+  pullPolicy: IfNotPresent
+
+ingress:
+  enabled: false
+
+service:
+  port: 30636
+  type: ClusterIP
+
+haproxy:
+  config: ldap-tls
+  ldapTls:
+    certificatePath: /etc/ssl/certs/ca-certificates.crt
+    #backend: ldap-example.org
+    frontend:
+      ports:
+        - name: frontend
+          number: 30636
+        - name: frontendTwo
```

Comment:
Would leave the second port out for the ldap-tls config as it only holds a single backend.